tests/test_epochs.py

```diff
@@ -0,0 +1,29 @@
+import ipfx.epochs as ep
+
+
+def test_find_stim_window():
```

Comment:
You should only test one case per test function.

Reply:
Agree, although you could keep this single function and use [pytest parameterization](https://docs.pytest.org/en/latest/example/parametrize.html) instead...

Reply:
I have changed to use parameterization in get_stim_characteristics function. The find_stim_window function may soon go away, so I did not touch the test